### DISCUSSION TITLE
2 boards cable level converter for sale Germany

### DISCUSSION THREAD

**Discussion by sandy1981Nett** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#147] [General]
![16368752649252487459115105491692](https://user-images.githubusercontent.com/87618510/141672055-62e8c6b5-2457-410e-a7fe-bd518b894e63.jpg)
2 boards cable level converter for sale in Germany

Christian

**Comment by GSzabados** [comment]
Damn, I should have looked here earlier. I've just ordered all the parts including the PCB(s).

**Comment by jarisiv** [comment]
@sandy1981Nett Hi, still available? Would be interested. Living in Finland.

**Comment by sandy1981Nett** [comment]
yes you still have here shipping shouldn't be a problem.

**Reply by jarisiv** [reply]
@sandy1981Nett Perfect! Can you send me an email with bank transfer details? I will reply to you with my home address. Many thanks!

**Reply by hacki11** [reply]
Still one left for shipment in Germany? @sandy1981Nett

**Reply by GSzabados** [reply]
@hacki11 - If @sandy1981Nett doesn't have it anymore, drop me a message.
I have ordered 5 PCBs back in December and all the necessary parts, ESP, cables/connectors, level converters, etc.
I still haven't built mine, but literally have 4 extra sets. 
I can send you one set from Poland. But about in 2 weeks time, because I am not there yet, where I have all the pieces.

**Reply by Xamude** [reply]
> Still one left for shipment in Germany? @sandy1981Nett

Just ordered Boards on Sunday to Germany. 
When they arrive i can send one Board to you

**Reply by sandy1981Nett** [reply]
hello, sorry i have not more board

**Reply by tedra86** [reply]
> @hacki11 - If @sandy1981Nett doesn't have it anymore, drop me a message. I have ordered 5 PCBs back in December and all the necessary parts, ESP, cables/connectors, level converters, etc. I still haven't built mine, but literally have 4 extra sets. I can send you one set from Poland. But about in 2 weeks time, because I am not there yet, where I have all the pieces.

@GSzabados Do you still have extra parts? (board, LLC, connectors)

**Reply by GSzabados** [reply]
@tedra86, yes, I still have. Where are you from?

**Reply by tedra86** [reply]
@GSzabados I'm from Hungary. Could you calculate a price for the PCB, LLC, connectors and shipping cost?

**Reply by GSzabados** [reply]
> I'm from Hungary.

Akkor keress meg a HA Community fórumon. És írj ott egy privát üzenet.
